frozen-abi/src/abi_example.rs

```diff
@@ -415,6 +415,13 @@ impl AbiExample for &Vec<u8> {
     }
 }
 
+impl AbiExample for &[u8] {
+    fn example() -> Self {
+        info!("AbiExample for (&[u8]): {}", type_name::<Self>());
+        &VEC_U8[..]
+    }
+}
+
```

Comment:
fyi, this impl will be replaced as par https://github.com/solana-labs/solana/pull/35118#discussion_r1494580278